samples/android/app/src/androidTest/java/org/mozilla/samples/gleancore/pings/SharedTestUtils.kt

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.gleancore.pings
+
+import org.json.JSONObject
+import org.mozilla.samples.gleancore.getPingServer
+import java.util.concurrent.TimeUnit
+
+/**
+ * Waits for ping with the given name to be received
+ * in the test ping server.
+ *
+ * @param pingName the name of the ping to wait for
+ * @param maxAttempts the maximum number of attempts
+ */
+fun waitForPingContent(
```

Comment:
I was doing something exactly like this to fix the android tests for #633 

I wonder if this should live in the test utils for glean core and not only samples.

Reply:
> I wonder if this should live in the test utils for glean core and not only samples.

Good call. @brizental would it be ok for you if I filed that as a follow-up bug?

Reply:
Of coruse, no worries. I might fix that very quickly as a part of #633, if you don't mind :)

Reply:
I don't :)